lib/turbo_boost/commands/runner.rb

```diff
@@ -256,14 +282,16 @@ def append_streams_to_response_body
     command_instance.turbo_streams.each { |stream| append_to_response_body stream }
   end
 
-  def append_command_token_to_response_body
-    append_to_response_body turbo_stream.invoke("TurboBoost.Commands.token=", args: [new_token], camelize: false)
-  rescue => error
-    Rails.logger.error "TurboBoost::Commands::Runner failed to append the Command token to the response! #{error.message}"
-  end
-
   def append_command_state_to_response_body
-    append_to_response_body turbo_stream.invoke("TurboBoost.State.initialize", args: [state.to_json, state.to_sgid_param], camelize: false)
+    # use the masked token for the client state
+    command_state[:command_token] = message_verifier.generate(new_command_token)
+    client_state = command_state.to_json
+
+    # use the unmasked token for the signed (server) state
+    command_state[:command_token] = new_command_token
+    signed_state = command_state.to_sgid_param
```

Comment:
hm, I think this would be cleaner if the `State` object just had two methods to render state

Reply:
Will punt on this refactor for now.